internal/runners/export/config.go

```diff
@@ -0,0 +1,96 @@
+package export
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+)
+
+// Filter is the --filter flag for the export config command, it implements captain.FlagMarshaler
+type Filter int
+
+const (
+	Unknown Filter = iota
+	Dir
+)
+
+var lookup = map[Filter]string{
+	Unknown: "unknown",
+	Dir:     "dir",
+}
+
+func (f Filter) String() string {
+	for k, v := range lookup {
+		if k == f {
+			return v
+		}
+	}
+	return lookup[Unknown]
+}
+
+func supportedFilters() []string {
+	var supported []string
+	for k, v := range lookup {
+		if k != Unknown {
+			supported = append(supported, v)
+		}
+	}
+
+	return supported
+}
+
+func SupportedFilters() string {
+	return strings.Join(supportedFilters(), ", ")
```

Comment:
This logic belongs in the cmdlet imo, should just return the slice of strings here.